.github/workflows/debian-tests.yml

```diff
@@ -0,0 +1,42 @@
+# Check if the binaries we have in this repository actually work when "npm i" is run
+name: Install and test the binary
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+#        - '14'
+        - '16'
+#        - '18'
```

Comment:
```suggestion
        node-version:
#        - '14'
        - '16'
#        - '18'
```

As we have the `container` matrix defined below I think we can just get rid of the above.